charts/nr-ebpf-agent/templates/nr-ebpf-agent-daemonset.yaml

```diff
@@ -10,8 +10,6 @@ metadata:
 spec:
   selector:
     matchLabels:
-      app: nr-ebpf-agent
```

Comment:
Do we need to remove these labels from the [nr-ebpf-agent-service.yaml](https://github.com/newrelic/helm-charts/blob/07a0fc171c3207df0571ca496f45c7626da329d7/charts/nr-ebpf-agent/templates/nr-ebpf-agent-service.yaml)? I believe the service/labels were added to configure TLS comm. between the agent/collector

Reply:
The other option is to rename the daemonset and make any necessary changes throughout the Helm chart to facilitate that change. 
However, considering that this is a daemonset and we're already using the selector 
`{{- include "newrelic.common.labels" . | nindent 4 }}` I do not believe we needed to include these selectors here. 
Also considering that the service that was created along with the implementation of TLS uses the selector `{{- include "newrelic.common.labels" . | nindent 4 }}` not  `app: nr-ebpf` is evidence that these labels/ selectors were added without any real functionality. 

Since both the service and this daemonset both use the selector 
`{{- include "newrelic.common.labels" . | nindent 4 }}` we should be good to delete this selector.